lib/ts/recipe/session/utils.ts

```diff
@@ -330,7 +331,7 @@ export async function updateClaimsInPayloadIfNeeded(
         logDebugMessage("updateClaimsInPayloadIfNeeded checking shouldRefetch for " + validator.id);
         if ("claim" in validator && (await validator.shouldRefetch(newAccessTokenPayload, userContext))) {
             logDebugMessage("updateClaimsInPayloadIfNeeded refetching " + validator.id);
-            const value = await validator.claim.fetchValue(this.getUserId(), userContext);
+            const value = await validator.claim.fetchValue(userId, userContext);
```

Comment:
Hmm.. that's weird. Why didn't TS complain about this.getUserId() not being present earlier?

Reply:
I'm not a 100% sure :/ But this actually shows something: we rarely need this for email verification (and that we need to add a specific test)